etrago/appl.py

```diff
@@ -124,6 +124,7 @@
         "cluster_foreign_AC": False,  # take foreign AC buses into account, True or False
         "method_gas": "kmedoids-dijkstra",  # choose clustering method: kmeans or kmedoids-dijkstra
         "n_clusters_gas": 14,  # total number of resulting CH4 nodes (DE+foreign)
+        "n_clusters_h2": 14,  # total number of resulting H2 nodes (DE+foreign)
```

Comment:
This needs to be changed in the args.json as well

Reply:
Changed in 2194e60ba83855ce305a0d886e27e39f62242913